### PR TITLE
yjw1029/add get_mind_data_set method

### PR DESCRIPTION
**Pull request by yjw1029**
### Description
Allow users to choose different MIND dataset using MIND_type parameter.

### Related Issues

https://github.com/microsoft/recommenders/issues/1178#issue-679468061

### Checklist:

- [x] I have followed the [contribution guidelines](../CONTRIBUTING.md) and code style for this project.
- [x] I have added tests covering my contributions.
- [x] I have updated the documentation accordingly.
- [x] This PR is being made to `staging` and not `master`.

### CODE REVIEW

**Comment by yjw1029** [comment]
Add feature enhancement issue about output MIND prediction.zip
https://github.com/microsoft/recommenders/issues/1180#issue-679980352

File: reco_utils/recommender/newsrec/newsrec_utils.py [hunk 1]
```diff
@@ -329,3 +329,40 @@ def newsample(news, ratio):
         return news + [0] * (ratio - len(news))
     else:
         return random.sample(news, ratio)
+
+
+def get_mind_data_set(type):
```
Comment (miguelgfierro) [inline reco_utils/recommender/newsrec/newsrec_utils.py:334]: there is already code for downloading and extracting mind https://github.com/microsoft/recommenders/blob/master/reco_utils/dataset/mind.py could you please use that instead?

Comment (miguelgfierro) [reply reco_utils/recommender/newsrec/newsrec_utils.py:334]: see an example of use here: https://github.com/microsoft/recommenders/blob/master/examples/02_model_content_based_filtering/dkn_deep_dive.ipynb

**Comment by miguelgfierro** [comment]
as discussed, we will merge this PR and review later

**Pull request merged**